<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MarkupsToSurface</class>
 <widget class="qMRMLWidget" name="MarkupsToSurface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>404</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="inputLabel">
       <property name="text">
        <string>Markups node:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="qMRMLNodeComboBox" name="inputSelector">
       <property name="toolTip">
        <string>Pick the input to the algorithm.</string>
       </property>
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLMarkupsShapeNode</string>
         <string>vtkMRMLMarkupsROINode</string>
         <string>vtkMRMLMarkupsFiducialNode</string>
        </stringlist>
       </property>
       <property name="showChildNodeTypes">
        <bool>false</bool>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>false</bool>
       </property>
       <property name="removeEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="outputModelLabel">
       <property name="text">
        <string>Model:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="qMRMLNodeComboBox" name="outputModelSelector">
       <property name="toolTip">
        <string>Create or replace the selected model.</string>
       </property>
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLModelNode</string>
        </stringlist>
       </property>
       <property name="hideChildNodeTypes">
        <stringlist notr="true"/>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="interactionNodeSingletonTag">
        <string notr="true"/>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="outputSegmentationLabel">
       <property name="text">
        <string>Segmentation:</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="qMRMLNodeComboBox" name="outputSegmentationSelector">
       <property name="toolTip">
        <string>Create or replace a segment in the selected segmentation.</string>
       </property>
       <property name="nodeTypes">
        <stringlist notr="true">
         <string>vtkMRMLSegmentationNode</string>
        </stringlist>
       </property>
       <property name="hideChildNodeTypes">
        <stringlist notr="true"/>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
       <property name="interactionNodeSingletonTag">
        <string notr="true"/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLineEdit" name="resultLineEdit">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="applyButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="toolTip">
      <string>Run the algorithm.</string>
     </property>
     <property name="text">
      <string>Apply</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>qMRMLWidget</class>
   <extends>QWidget</extends>
   <header>qMRMLWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>MarkupsToSurface</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>inputSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>122</x>
     <y>132</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>61</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarkupsToSurface</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputModelSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>69</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>MarkupsToSurface</sender>
   <signal>mrmlSceneChanged(vtkMRMLScene*)</signal>
   <receiver>outputSegmentationSelector</receiver>
   <slot>setMRMLScene(vtkMRMLScene*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>176</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
